### ISSUE TITLE
304 Database files extension

### ISSUE THREAD

**Issue by lennybacon** [dirceuresende/checklist_seguranca#5]
> This setting validates whether SQL Server is using the default extensions for data files, logs, and backups, which are targets of Ransomwares, such WannaCry

I understand the action but doesn't it fall into the category **Security by obscurity**?

If an attacker has found its way onto the SQL Server machine and has access to the files he also has access to SQL server and can enumerate the databases and their files...

**Comment by dirceuresende** [comment]
I agree with you. This item is more suitable to "Security by obscurity" category, but I don't have this category yet.. lol

This item aims some ransomwares, like wannacry, that only tries to encrypt some pre-defined file extensions. It's not the best solution, mas it helps in some cases.
